eslint.config.mjs

```diff
@@ -37,13 +53,15 @@
         "no-undef-init": "off",
         "newline-per-chained-call": "off",
         "no-useless-escape": "off",
+        "no-redeclare":"off",
```

Comment:
Ideally we should not add any custom rule. if some rules do not make sense for our repos, better to update the guidelines repo. maybe worth a follow up ticket? (to remove all these custom rules)?